editor/src/java/com/defold/editor/Splash.java

```diff
@@ -81,59 +81,25 @@ private static String randomElement(List<String> tips) {
     }
 
     private static void randomGame(Scene scene) {
-        String imagePath = "";
-        String gameName = "";
-        switch(randomInt(10)) {
```

Comment:
10 isn't included, so it's never showed

I changed it to make sure we avoid such mistakes in the future